src/boxes/extensions/box.scss

```
@@ -20,4 +20,4 @@
   @include brand-01(background-color);
   color: white;
   padding: 8px;
```

```suggestion
  padding: layout.$spacing-03;
```
Minor nitpick, shall we change this to use carbon spacing, we'd have to use carbon's layout

---

`layout` and `spacing` are aliases of one another... It's the same underlying tokens.